app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -25,6 +23,16 @@
   padding-left: $cads-spacing-3;
 }
 
+.cads-table tr:first-child th:nth-of-type(3),
+.cads-table tr:first-child th:nth-of-type(2) {
+  width: 125px;
```

Comment:
Is the width added so that the rating doesn't go into two lines? If so, then it can be adjusted by setting `white-space: nowrap;` to the table cells without setting width. jIt should probably also be applied to other cells which contain ratings.